code/index.html

```diff
@@ -8,20 +8,32 @@
     <link
       href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
       rel="stylesheet" />
-    <title>Chat Bot</title>
+      <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Comforter&display=swap"/>
+    <!-- FontAwesome link -->
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>Hairdo Bot</title>
   </head>
 
   <body>
-    <h1>Welcome to xxx.</h1>
-    <h2>Ask about xxxx.</h2>
+    <header>
+      <img src="assets\heroimage.jpg" alt="">
```

Comment:
Don't forget to fill in the alt-text here, for SEO and for people with visual impairment, or in case the picture don't work